scripts/build.js

```diff
@@ -0,0 +1,66 @@
+const metalsmith  = require('metalsmith')
+const inplace     = require('metalsmith-in-place')   // render templating syntax in your source files
+const layouts     = require('metalsmith-layouts')    // apply layouts to your source files
+const markdown    = require('metalsmith-markdown')   // convert markdown files to html
+const metallic    = require('metalsmith-metallic')   // highlight code in markdown files
+const permalinks  = require('metalsmith-permalinks') // apply a permalink pattern to files
+const sass        = require('metalsmith-sass')       // convert Sass files to CSS
+const rollup      = require('metalsmith-rollup')     // compile and bundle JS files
+
+const paths       = require('../config/paths.json')
+
+const views = [
+  paths.layouts,
+  paths.partials,
+  paths.components,
+  paths.govukfrontend
+]
+
+metalsmith(__dirname)
+  .metadata({
+    title: "GOV.UK Design System"
+  })
+  .source('../' + paths.source)
+  .destination('../' + paths.public)
+  .clean(true)
+  .use(sass({
+    files:['../source/stylesheets/main.scss'],
+    includePaths:['node_modules'],
+    outputDir: 'stylesheets',
+  }))
+  .use(rollup({
+    entry: 'source/javascripts/vendor/modernizr.js',
+    dest: 'javascripts/application.js',
+  }))
+  .use(rollup({
+    entry: 'source/javascripts/vendor/html5shiv-printshiv.js',
+    dest: 'javascripts/ie.js',
+  }))
+  .use(inplace({
```

Comment:
Comment suggestion: _Compile nunjucks inside Readme files and all non-layout views_